prprocessor/__main__.py

```diff
@@ -94,6 +110,15 @@ class Config:
 logger = logging.getLogger('prprocessor')  # pylint: disable=invalid-name
 
 
+def is_commit_exempt(commit: Commit, exempt_patterns: list) -> bool:
+    """Check if commit subject matches any exemption pattern."""
+    import re
+    for pattern in exempt_patterns:
+        if re.search(pattern, commit.subject, re.IGNORECASE):
+            return True
+    return False
```

Comment:
Maybe a bit of code golfing
```suggestion
    return any(re.search(pattern, commit.subject, re.IGNORECASE) for pattern in exempt_patterns)
```

Reply:
I was thinking about that too.